.gitignore

```diff
@@ -422,6 +422,10 @@ dist
 # TernJS port file
 .tern-port
 
+# VS-Code launch.json file for cmd line argument debugging and its binary
```

Comment:
Tbh I think I'd be more in favor of committing a solid `launch.json` file so that we everyone can debug with VSCode (I always hate setting that up myself lol)

Reply:
Ok sweet I will commit what I have for now. I have one configuration for our `dump` command which lets you debug dictionary dumps via what's in our `examples` folder 💪🏻

Reply:
Also I'll chuck one in for creating an odict file